<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 11">
<meta name=Originator content="Microsoft Word 11">
<link rel=File-List href="screenshots_files/filelist.xml">
<link rel=Edit-Time-Data href="screenshots_files/editdata.mso">
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
w\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
</style>
<![endif]-->
<title>Screenshots of STEM</title>
<!--[if gte mso 9]><xml>
 <o:DocumentProperties>
  <o:Author>Jason</o:Author>
  <o:LastAuthor>Jason</o:LastAuthor>
  <o:Revision>2</o:Revision>
  <o:TotalTime>212</o:TotalTime>
  <o:LastPrinted>2004-11-02T19:46:00Z</o:LastPrinted>
  <o:Created>2006-04-27T22:53:00Z</o:Created>
  <o:LastSaved>2006-04-27T22:53:00Z</o:LastSaved>
  <o:Pages>1</o:Pages>
  <o:Words>1158</o:Words>
  <o:Characters>6607</o:Characters>
  <o:Company> </o:Company>
  <o:Lines>55</o:Lines>
  <o:Paragraphs>15</o:Paragraphs>
  <o:CharactersWithSpaces>7750</o:CharactersWithSpaces>
  <o:Version>11.6360</o:Version>
 </o:DocumentProperties>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:WordDocument>
  <w:PunctuationKerning/>
  <w:ValidateAgainstSchemas/>
  <w:SaveIfXMLInvalid>false</w:SaveIfXMLInvalid>
  <w:IgnoreMixedContent>false</w:IgnoreMixedContent>
  <w:AlwaysShowPlaceholderText>false</w:AlwaysShowPlaceholderText>
  <w:Compatibility>
   <w:SelectEntireFieldWithStartOrEnd/>
   <w:UseWord2002TableStyleRules/>
  </w:Compatibility>
  <w:BrowserLevel>MicrosoftInternetExplorer4</w:BrowserLevel>
 </w:WordDocument>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <w:LatentStyles DefLockedState="false" LatentStyleCount="156">
 </w:LatentStyles>
</xml><![endif]-->
<style>
<!--
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-parent:"";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	tab-stops:center 3.0in right 6.0in;
	font-size:12.0pt;
	font-family:"Times New Roman";
	mso-fareast-font-family:"Times New Roman";}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;
	text-underline:single;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;
	text-underline:single;}
 /* Page Definitions */
 @page
	{mso-footnote-separator:url("screenshots_files/header.htm") fs;
	mso-footnote-continuation-separator:url("screenshots_files/header.htm") fcs;
	mso-endnote-separator:url("screenshots_files/header.htm") es;
	mso-endnote-continuation-separator:url("screenshots_files/header.htm") ecs;}
@page Section1
	{size:8.5in 11.0in;
	margin:1.0in 1.25in 1.0in 1.25in;
	mso-header-margin:.5in;
	mso-footer-margin:.5in;
	mso-even-footer:url("screenshots_files/header.htm") ef1;
	mso-footer:url("screenshots_files/header.htm") f1;
	mso-paper-source:0;}
div.Section1
	{page:Section1;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-parent:"";
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-para-margin:0in;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Times New Roman";
	mso-ansi-language:#0400;
	mso-fareast-language:#0400;
	mso-bidi-language:#0400;}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="2050"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US link=blue vlink=purple style='tab-interval:.5in'>

<div class=Section1>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:18.0pt'>Screenshots of the <a
href="http://www.cs.cmu.edu/~jernst/stem">Short Time-series Expression Miner</a>
(STEM)<o:p></o:p></span></p>

<p class=MsoNormal align=center style='text-align:center'><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><span style='mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="_x0000_i1025" type="#_x0000_t75" style='width:6in;
 height:336pt'>
 <v:imagedata src="screenshots_files/image001.png" o:title="inputstem"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=448
src="screenshots_files/image002.gif" v:shapes="_x0000_i1025"><![endif]></span></p>

<p class=MsoNormal>The input interface above is the first screen that appears
when <a href="http://www.cs.cmu.edu/~jernst/stem">STEM</a> is executed.<span
style='mso-spacerun:yes'>  </span>In the first section of this interface a user
specifies the input data and normalization options.<span
style='mso-spacerun:yes'>  </span>Repeat data can be specified through a dialog
window that appears by clicking on the “Repeat Data” button.<span
style='mso-spacerun:yes'>  </span>In the second section of the interface a user
specifies the gene annotation source which can be any gene annotation source
available from <a
href="http://www.geneontology.org/GO.current.annotations.shtml">http://www.geneontology.org/GO.current.annotations.shtml</a>
or it can be provided by the user.<span style='mso-spacerun:yes'>  </span>In the
third section of the interface a user can choose to either use the STEM
clustering method, a novel clustering method specifically for short time series
gene expression data (Ernst et al, Bioinformatics 2005), or the standard
K-means clustering algorithm.<span style='mso-spacerun:yes'> 
</span>Screenshots on this page show images for the STEM clustering
method.<span style='mso-spacerun:yes'>   </span>When the STEM Clustering method
is selected two parameters, the number of model profiles and the maximum unit
change in model profiles between two time points, are available on this screen
as shown above.<span style='mso-spacerun:yes'>  </span>A user can click “Advanced
Options” to change more advance parameters.<span style='mso-spacerun:yes'> 
</span>Clicking “Execute” executes the clustering algorithm and brings up the
main window of the software to display the results.<span
style='mso-spacerun:yes'>  </span>By clicking on any question mark a user can
receive help information about the corresponding input field.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1027" type="#_x0000_t75"
 style='width:6in;height:231pt'>
 <v:imagedata src="screenshots_files/image003.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=308
src="screenshots_files/image004.jpg" v:shapes="_x0000_i1027"><![endif]></p>

<p class=MsoNormal>Repeat data of an experiment can be loaded from a dialog
window as above.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1028" type="#_x0000_t75"
 style='width:431.25pt;height:146.25pt'>
 <v:imagedata src="screenshots_files/image005.png" o:title="filteringv111"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=575 height=195
src="screenshots_files/image006.gif" v:shapes="_x0000_i1028"><![endif]></p>

<p class=MsoNormal>Advanced options related to filtering genes.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1029" type="#_x0000_t75"
 style='width:431.25pt;height:146.25pt'>
 <v:imagedata src="screenshots_files/image007.png" o:title="modelprofilev111"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=575 height=195
src="screenshots_files/image008.gif" v:shapes="_x0000_i1029"><![endif]></p>

<p class=MsoNormal>Advanced options related to selecting model profiles and assessing
their significance.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1030" type="#_x0000_t75"
 style='width:431.25pt;height:124.5pt'>
 <v:imagedata src="screenshots_files/image009.png" o:title="clusteringv11"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=575 height=166
src="screenshots_files/image010.gif" v:shapes="_x0000_i1030"><![endif]></p>

<p class=MsoNormal>Advanced options related to clustering significant model
profiles.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1031" type="#_x0000_t75"
 style='width:431.25pt;height:124.5pt'>
 <v:imagedata src="screenshots_files/image011.png" o:title="geneannotationsv11"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=575 height=166
src="screenshots_files/image012.gif" v:shapes="_x0000_i1031"><![endif]></p>

<p class=MsoNormal>Advanced options related to gene annotations.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1032" type="#_x0000_t75"
 style='width:431.25pt;height:124.5pt'>
 <v:imagedata src="screenshots_files/image013.png" o:title="goanalysisv11"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=575 height=166
src="screenshots_files/image014.gif" v:shapes="_x0000_i1032"><![endif]></p>

<p class=MsoNormal>Advanced options related to the Gene Ontology enrichment analysis.
</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1033" type="#_x0000_t75"
 style='width:6in;height:324pt'>
 <v:imagedata src="screenshots_files/image015.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=432
src="screenshots_files/image016.jpg" v:shapes="_x0000_i1033"><![endif]></p>

<p class=MsoNormal>Above is main window to the software displaying results for
the STEM clustering method.<span style='mso-spacerun:yes'>  </span>Each gene is
assigned to the model profile its temporal expression profile most closely
matches.<span style='mso-spacerun:yes'>  </span>The profiles that have a
statistically significant number of genes assigned based on a permutation test
are shaded.<span style='mso-spacerun:yes'>  </span>Statistically significant
profiles that are similar form a cluster of profiles, and have the same
color.<span style='mso-spacerun:yes'>  </span>The number in the top left hand
corner of each profile box is the profile ID.<span style='mso-spacerun:yes'> 
</span>One can reorder the profiles on this screen by selecting “Order Profiles
By” or reorder the clusters of profiles by selecting “Order Clusters By.”<span
style='mso-spacerun:yes'>  </span>One can see a table of all genes passing
filter and the profiles to which they were assigned by clicking on the “Main
Gene Table” button.<span style='mso-spacerun:yes'>  </span>The “Filtered Gene
List” button displays a table with those genes that did not pass filter.<span
style='mso-spacerun:yes'>  </span>The “Compare” option for comparing two data
sets under different experimental conditions is described at <a
href="http://www.cs.cmu.edu/~jernst/stem/comparison.html">http://www.cs.cmu.edu/~jernst/stem/comparison.html</a>.</p>

<span style='font-size:12.0pt;font-family:"Times New Roman";mso-fareast-font-family:
"Times New Roman";mso-ansi-language:EN-US;mso-fareast-language:EN-US;
mso-bidi-language:AR-SA'><br clear=all style='page-break-before:always'>
</span>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1034" type="#_x0000_t75"
 style='width:388.5pt;height:321.75pt'>
 <v:imagedata src="screenshots_files/image017.png" o:title="orderby"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=518 height=429
src="screenshots_files/image018.gif" v:shapes="_x0000_i1034"><![endif]></p>

<p class=MsoNormal>After clicking the “Order Profiles By” button the above
dialog appears.<span style='mso-spacerun:yes'>  </span>The profiles can be
reordered by enrichment of genes from any GO category or a user defined set, by
ID, by the p-value significance of the number of genes assigned being greater
than the expected number based on a permutation test, or the number of genes
assigned or expected.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1035" type="#_x0000_t75"
 style='width:6in;height:324pt'>
 <v:imagedata src="screenshots_files/image019.png" o:title="mainsort"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=432
src="screenshots_files/image020.gif" v:shapes="_x0000_i1035"><![endif]></p>

<p class=MsoNormal>The profiles are reordered based on enrichment of cell cycle
genes.<span style='mso-spacerun:yes'>  </span>The profiles most enriched for
the cell cycle appear to the left of the top row.<span
style='mso-spacerun:yes'>   </span>The number of cell cycle genes and the
p-value enrichment of genes assigned to the profile appear in the lower left
corner of the profile box.<span style='mso-spacerun:yes'>  </span>The p-value
of enrichment for profiles can be based on either the actual number of genes
assigned to the profile or the expected number.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1036" type="#_x0000_t75"
 style='width:449.25pt;height:322.5pt'>
 <v:imagedata src="screenshots_files/image021.png" o:title="orderclust"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=599 height=430
src="screenshots_files/image022.gif" v:shapes="_x0000_i1036"><![endif]></p>

<p class=MsoNormal>The clusters of profiles can also be reordered based on
their importance to a particular GO Category, in the image above it is the cell
cycle.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1037" type="#_x0000_t75"
 style='width:6in;height:315.75pt'>
 <v:imagedata src="screenshots_files/image023.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=421
src="screenshots_files/image024.jpg" v:shapes="_x0000_i1037"><![endif]></p>

<p class=MsoNormal>Pressing the help icon on the main window with all the
profiles displays a legend explaining what each number means on a profile box.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1038" type="#_x0000_t75"
 style='width:357.75pt;height:345.75pt'>
 <v:imagedata src="screenshots_files/image025.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=477 height=461
src="screenshots_files/image026.jpg" v:shapes="_x0000_i1038"><![endif]></p>

<p class=MsoNormal>A user can also reorder profiles or clusters of profiles
based on a user defined set of genes through the above dialog box.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1039" type="#_x0000_t75"
 style='width:6in;height:167.25pt'>
 <v:imagedata src="screenshots_files/image027.png" o:title="zoom"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=223
src="screenshots_files/image028.gif" v:shapes="_x0000_i1039"><![endif]></p>

<p class=MsoNormal>The main screen with all the model profiles is zoomable and
pannable.<span style='mso-spacerun:yes'>  </span>One can zoom in or out on any
portion of the screen.<span style='mso-spacerun:yes'>  </span>The ability to
zoom and pan is powered by the open source software, Piccolo, available from <a
href="http://www.cs.umd.edu/hcil/piccolo/">http://www.cs.umd.edu/hcil/piccolo/</a>.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1040" type="#_x0000_t75"
 style='width:6in;height:255.75pt'>
 <v:imagedata src="screenshots_files/image029.png" o:title=""/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=341
src="screenshots_files/image030.jpg" v:shapes="_x0000_i1040"><![endif]></p>

<p class=MsoNormal>A user can see a table of all genes passing filter and the
profile to which they were assigned.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1041" type="#_x0000_t75"
 style='width:342pt;height:256.5pt'>
 <v:imagedata src="screenshots_files/image031.png" o:title="p43"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=456 height=342
src="screenshots_files/image032.gif" v:shapes="_x0000_i1041"><![endif]></p>

<p class=MsoNormal>By clicking on a profile box on the main screen detailed
information about the profile is displayed.<span style='mso-spacerun:yes'> 
</span>The above image shows an example of such a window.<span
style='mso-spacerun:yes'>  </span>The detailed information about the profile
includes a graph of the expression pattern of genes assigned to the profile,
how many genes were assigned, how many genes were expected, and the associated
p-value significance of the number genes assigned being greater than what was
expected.<span style='mso-spacerun:yes'>  </span>A user also has the option
from this window to display a table with all the genes assigned to the profile
by pressing the “Profile Gene Table” or a table with a Gene Ontology (GO)
enrichment analysis results for the set of genes assigned to the profile by
pressing “Profile GO Table.”<span style='mso-spacerun:yes'>  </span>If a
profile has a statistically significant number of genes assigned and was
grouped in a cluster with another profile then as above the “Cluster Gene
Table” and “Cluster GO Table” buttons also appear.<span
style='mso-spacerun:yes'>  </span>These buttons are analogous to the “Profile
Gene Table” and “Profile GO Table” buttons, but the gene table and GO
enrichment analysis include all genes assigned to any profile in the cluster of
profiles to which the displayed profile belongs.<span
style='mso-spacerun:yes'>  </span>Profiles of the same color in the main window
belong to the same cluster of profiles.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><span style='mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="_x0000_i1042" type="#_x0000_t75" style='width:3in;height:162pt'>
 <v:imagedata src="screenshots_files/image033.png" o:title="p9reg"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=288 height=216
src="screenshots_files/image034.gif" v:shapes="_x0000_i1042"><![endif]><!--[if gte vml 1]><v:shape
 id="_x0000_i1043" type="#_x0000_t75" style='width:3in;height:162pt'>
 <v:imagedata src="screenshots_files/image035.png" o:title="p9cell"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=288 height=216
src="screenshots_files/image036.gif" v:shapes="_x0000_i1043"><![endif]></span></p>

<p class=MsoNormal>If the profiles on the main screen are ordered based on
enrichment for a GO Category or gene set, the dialog window with detailed
information about the profile will also include an option to plot just the
genes belonging to the GO Category or gene set by which the profiles are
ordered.<span style='mso-spacerun:yes'>  </span>The image on left shows the
window that appears when a user clicks on the monotonically decreasing profile
when the profiles are reordered based on the cell cycle.<span
style='mso-spacerun:yes'>  </span>Pressing the button “Click to plot only
profile cell cycle genes” on the window on left replots the window with only
those genes assigned to the profile that are also annotated as being cell cycle
genes as shown on right.<span style='mso-spacerun:yes'>  </span>There is also
the option through the button, “Profile cell cycle Gene Table,” to display a
table containing just the cell cycle genes assigned to the profile.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1044" type="#_x0000_t75"
 style='width:6in;height:139.5pt'>
 <v:imagedata src="screenshots_files/image037.png" o:title="genetable"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=186
src="screenshots_files/image038.gif" v:shapes="_x0000_i1044"><![endif]></p>

<p class=MsoNormal>Above is an example of a profile gene table.<span
style='mso-spacerun:yes'>  </span>This table contains all genes assigned to the
monotonically decreasing profile. </p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_s1029" type="#_x0000_t75"
 style='width:6in;height:266.2pt;mso-position-horizontal-relative:char;
 mso-position-vertical-relative:line' fillcolor="#bbe0e3">
 <v:imagedata src="screenshots_files/image039.png" o:title=""/>
 <w:wrap type="none"/>
 <w:anchorlock/>
</v:shape><![endif]--><![if !vml]><img width=576 height=355
src="screenshots_files/image040.jpg" v:shapes="_x0000_s1029"><![endif]></p>

<p class=MsoNormal>Above is an example of a Gene Ontology (GO) enrichment
analysis table for a profile, in this case the monotonically decreasing
profile.<span style='mso-spacerun:yes'>  </span>The expression profile is
likely relevant to categories for which the p-value is low.<span
style='mso-spacerun:yes'>   </span>By default the enrichment analysis is based
on a profile’s actual size, that is the number of genes assigned to the
profile.<span style='mso-spacerun:yes'>  </span>A table can also be displayed
with the enrichment analysis based on a profile’s expected size by pressing the
“Click for GO Results Based on the Profile’s Expected Size.”<span
style='mso-spacerun:yes'>  </span>Clicking on a row of the table opens a new
table showing all genes assigned to the profile that also are annotated as
belonging to the GO category.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><!--[if gte vml 1]><v:shape id="_x0000_i1045" type="#_x0000_t75"
 style='width:6in;height:145.5pt'>
 <v:imagedata src="screenshots_files/image041.png" o:title="cellcycle"/>
</v:shape><![endif]--><![if !vml]><img border=0 width=576 height=194
src="screenshots_files/image042.gif" v:shapes="_x0000_i1045"><![endif]></p>

<p class=MsoNormal>Above is a table displaying just the cell cycle genes
assigned to the monotonically decreasing profile.</p>

</div>

</body>

</html>
