<html>
<head>
<title>Clustering Short Time Series Gene Expression Data
</title>
</head>

<BODY BGCOLOR="#FFFFFF" VLINK="#0000FF"><P> 

<P> 
<P>&nbsp; 
<TABLE BORDER=0>
  <TR> 
    <TD> <CENTER>
        <FONT SIZE="+3" 
FACE="Helvetica" COLOR="#AF0000"><B>Clustering Short Time
Series Gene Expression Data</B></FONT> 
      </CENTER></TD>
  </TR>
</TABLE>
<i>This webpage provides supporting information for the ISMB 2005 
paper</i>:
<p>
<a href="http://www.cs.cmu.edu/~jernst">J. Ernst</a>,
<a href="http://www.mgb.pitt.edu/personnel/view.asp?uid=gjnau"> G.J. Nau</a>, and <a 
href="http://www.cs.cmu.edu/~zivbj">Z. Bar-Joseph</a><br>
<b><a href="https://academic.oup.com/bioinformatics/article/21/suppl_1/i159/203147">
Clustering Short Time Series Gene Expression Data </a href>
</b><br>
<i>Bioinformatics (Proceedings of ISMB 2005)</i>, 21 Suppl. 1, pp. i159-i168, 2005. 
<br><br>

<b>Abstract</b><br>
<i>Motivation:</i>
Time series expression experiments are used to study a
wide range of biological systems. More than 80%
of all time series expression datasets are short (8 time points or fewer).
These datasets present unique challenges. Due to the large number
of genes profiled (often tens of thousands) and the small number
of time points many patterns are expected to arise at random. Most
clustering algorithms are unable to distinguish between real and
random patterns. 
<br>
<br>
<i>Results:</i>
We present an algorithm specifically designed for clustering
short time series expression data.
Our algorithm works by assigning genes to a pre-defined
set of model profiles that capture the potential distinct patterns
that can be expected from the experiment. We discuss how to obtain such a
set of profiles and how to determine the significance of each of these
profiles. Significant profiles are retained for further analysis and can be 
combined to form clusters. 
We tested our method on both simulated and real biological data. Using
immune response data we show that our algorithm can correctly detect
the temporal profile of relevant functional categories.
Using GO analysis we show that our
algorithm outperforms both
general clustering algorithms and algorithms designed specifically for
clustering time series gene expression data.  

<P> 
<HR>
<TABLE BORDER=0 WIDTH=10000>
  <TR> 
    <TD WIDTH=1000 height="471" 
VALIGN=top> 
      <P><a href="https://ernstlab.github.io/STEM"><FONT SIZE="+1" COLOR="#AF0000"><B>
Software - STEM: Short Time-series Expression Miner</A></B></FONT></P>
      <P><a href="biodata.html"><FONT SIZE="+1" COLOR="#AF0000"><B>
Biological Data</A></B></FONT></P>
      <P><a href="simulated.html"><FONT SIZE="+1" COLOR="#AF0000"><B>
Simulated Data</A></B></FONT></P>
<P><a href="supporting.pdf"><FONT SIZE="+1" COLOR="#AF0000"><B>
Supporting Lemmas and Theorems </A></B></FONT></P>
      <P><a href="robust.html"><FONT SIZE="+1" COLOR="#AF0000"><B>
Additional Experimental Results - Robustness Analysis for the Maximum 
Unit Change Parameter (<i>c</i>) 
</A></B></FONT></P>
      <P><a href="kmeans2.html"><FONT SIZE="+1" COLOR="#AF0000"><B>
Additional Experimental Results - Comparison with <i>k</i>-means with 
<i>k</i>=2</A></B></FONT></P>
<TD WIDTH=450 height="471"> 
</TABLE>

<!-- Start of StatCounter Code -->
<script type="text/javascript" language="javascript">
var sc_project=621057; 
var sc_invisible=1; 
var sc_partition=3; 
var sc_security="eb5113cd"; 
var sc_remove_link=1; 
</script>

<script type="text/javascript" language="javascript" 
src="http://www.statcounter.com/counter/counter.js"></script><noscript><img  
src="http://c4.statcounter.com/counter.php?sc_project=621057&amp;java=0&amp;security=eb5113cd&amp;invisible=1" 
alt="free statistics" border="0"> </noscript>
<!-- End of StatCounter Code -->


</body>

</html>
