<html>
<head>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-28866752-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>

<Title>ChromHMM: Chromatin state discovery and
characterization</Title>
</head>

<body>
<h1><center>ChromHMM: Chromatin state discovery and 
characterization</center></h1>
<center>
<img src="segments.png" width=600 height=200>
</center>
ChromHMM is software for learning and characterizing chromatin states. 
ChromHMM can integrate multiple chromatin datasets such as ChIP-seq data 
of various histone modifications to discover de novo the major 
re-occuring combinatorial and spatial patterns of marks. ChromHMM is 
based on a multivariate Hidden Markov Model that explicitly models the 
presence or absence of each chromatin mark. The resulting model can then 
be used to systematically annotate a genome in one or more cell types. 
By automatically computing state enrichments for large-scale functional 
and annotation datasets ChromHMM facilitates the biological 
characterization of each state. ChromHMM also produces files with 
genome-wide maps of chromatin state annotations that can be directly 
visualized in a genome browser. 
<br>

<br>
<li><a href="ChromHMM.zip" onClick="javascript: _gaq.push(['_trackPageview', '/downloads/ChromHMM.zip']);">ChromHMM software 
v1.25</a> (<a href="versionlog.txt">version log</a>)<br>
<li> <a href="ChromHMM_manual.pdf">ChromHMM manual</a><br>

<br>

Quick instructions on running ChromHMM:<br>
1. Install Java 1.7 or later if not already installed. <br>
2. Unzip the file ChromHMM.zip<br>
3. To try out ChromHMM learning a 10-state model on the sample data
enter from a command line in the directory with the ChromHMM.jar file the command:<br><br>
java -mx1600M -jar ChromHMM.jar LearnModel SAMPLEDATA_HG18 OUTPUTSAMPLE 10 hg18
<br><br>
After termination in ~5-10 minutes a file in OUTPUTSAMPLE/webpage_10.html will be created showing 
output images and  linking to all the output files created. If a web browser is found on the 
computer the webpage will automatically be opened in it.<br> In general binarized input for the <i>LearnModel</i> command can be generated 
by first running the <i>BinarizeBed</i> command on bed files with coordinates of aligned reads or the <i>BinarizeBam</i> command on
bam files with the coordinates of aligned reads.<br><br>

<li>The ChromHMM software is described in:<br>
Ernst J, Kellis M.
<a href="http://www.nature.com/nmeth/journal/v9/n3/full/nmeth.1906.html">
ChromHMM: automating chromatin-state discovery and characterization</a></b>. <i>Nature Methods</i>,
9:215-216, 2012.<br>
<li> A protocols paper on using ChromHMM is available here:<br>
Ernst J, Kellis M.
<a href="https://www.nature.com/articles/nprot.2017.124">Chromatin-state discovery and genome annotation with ChromHMM</a>.
<i>Nature Protocols</i>, 12:2478-2492, 2017.<br> 
<li> Here are links to some existing ChromHMM annotations in hg19 available for <a href="http://compbio.mit.edu/roadmap">127 Reference Epigenomes (Roadmap Epigenomics)</a>,
 <a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?g=wgEncodeBroadHmm&db=hg19">9-ENCODE cell types (from Ernst et al, Nature 2011)</a>, and 
<a href="https://genome.ucsc.edu/cgi-bin/hgTrackUi?g=wgEncodeAwgSegmentation&db=hg19">6-ENCODE cell types (from ENCODE Integrative Analysis)</a>.<br>
<li> A liftover of the hg19 annotations to hg38 for the 127 Reference Epigenomes (Roadmap Epigenomics) is available <a href="http://compbio.mit.edu/roadmap">here</a>.<br>
<li> <a href="https://github.com/ernstlab/full_stack_ChromHMM_annotations">ChromHMM annotations based on a full stack model of the Roadmap Epigenomics
data</a> providing a universal chromatin state annotation of the human genome is described in:<br>
Vu H, Ernst J.
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02572-z">
Universal annotation of the human genome through integration of over a thousand epigenomic datasets.</a>
<i>Genome Biology</i>, 23:9, 2022.<br>
and <a href="https://github.com/ernstlab/mouse_fullStack_annotations">an analogous annotations for mouse based on ENCODE data</a> is also available and
described in: <br>
Vu H, Ernst J.
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-023-02994-x">
Universal chromatin state annotation of the mouse genome.</a>
<i>Genome Biology</i>, 24:153, 2023.<br>




<li>Contact Jason Ernst (jason.ernst at ucla dot edu) with any questions, 
comments, or bug reports.
<li>Subscribe to a <a 
href="http://groups.google.com/group/ChromHMM-announce/"> mailing 
list for announcements of new versions</a>
<br>
<li>ChromHMM is released under a <a href="http://www.gnu.org/licenses/gpl-3.0.html">GPL 3 license</a>. <br>
<li>ChromHMM source code is available on GitHub <a href="https://github.com/jernst98/ChromHMM">here</a>.<br>
<li>Funding for ChromHMM provided by NSF Postdoctoral Fellowship 
0905968 to 
JE and grants from the National Institutes of Health (NIH 
1-RC1-HG005334 
and NIH 1 U54 HG004570).
</body>
</html>
